2020-12-15 18:23:17,572 - root - INFO - main.py:63 - START SERVER
2020-12-15 18:23:18,399 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 18:23:18,442 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:23:18,463 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:23:18,565 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:23:18,583 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:23:19,219 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 18:23:19,314 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 18:30:50,887 - root - INFO - main.py:63 - START SERVER
2020-12-15 18:30:51,547 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 18:30:51,567 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:30:51,576 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:30:51,610 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:30:52,269 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 18:30:52,323 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 18:31:01,027 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:31:10,278 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:32:30,498 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:32:40,996 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:33:02,967 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:33:10,270 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:33:12,315 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:33:20,688 - root - ERROR - task_controler.py:169 - a bytes-like object is required, not 'dict'
2020-12-15 18:33:44,922 - root - INFO - main.py:63 - START SERVER
2020-12-15 18:33:45,521 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 18:33:45,535 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:33:45,541 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:33:45,549 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:33:46,462 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 18:33:46,521 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 18:33:50,183 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:33:50,415 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:33:50,616 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:33:50,813 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:33:51,058 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:00,208 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:00,411 - root - ERROR - task_controler.py:170 - a bytes-like object is required, not 'dict'
2020-12-15 18:34:00,417 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:00,605 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:00,789 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:00,971 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:01,157 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:10,218 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:10,436 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:10,632 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:10,814 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:11,001 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:20,210 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:20,428 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:20,640 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:20,833 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:21,026 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:30,208 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:30,405 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:30,615 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:30,823 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:31,016 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:40,170 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:40,396 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:40,582 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:40,759 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:40,937 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:41,112 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:50,195 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:50,389 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:50,592 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:50,794 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:34:51,013 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:00,208 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:00,414 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:00,612 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:00,804 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:00,993 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:01,362 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:10,195 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:10,398 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:10,595 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:10,772 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:10,953 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:11,145 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:20,215 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:20,424 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:20,627 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:20,839 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:21,033 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:30,184 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:30,417 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:30,637 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:30,817 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:30,990 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:31,226 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:40,205 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:40,400 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:40,579 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:40,820 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:41,023 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:50,185 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:50,359 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:50,582 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:50,782 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:50,974 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:51,227 - root - ERROR - Object2.py:62 - '03'
2020-12-15 18:35:59,931 - root - INFO - main.py:63 - START SERVER
2020-12-15 18:36:00,538 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 18:36:00,647 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:36:00,660 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:36:00,765 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:36:01,005 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 18:36:01,046 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 18:38:48,781 - root - INFO - main.py:63 - START SERVER
2020-12-15 18:38:49,378 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 18:38:49,395 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:38:49,403 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:38:49,424 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:38:49,434 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:38:49,917 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 18:38:49,951 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 18:40:00,152 - root - ERROR - task_controler.py:171 - [Errno 32] Broken pipe
2020-12-15 18:40:07,013 - root - INFO - main.py:63 - START SERVER
2020-12-15 18:40:07,669 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 18:40:07,684 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:40:07,697 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:40:08,010 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 18:40:08,019 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 18:41:50,942 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:42:10,041 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:42:20,151 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:42:30,508 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:42:50,341 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:43:03,806 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:43:11,004 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:43:13,057 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:43:20,208 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:43:23,265 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:45:03,054 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:45:13,577 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:45:41,814 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 18:47:13,039 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:47:23,504 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:48:13,299 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:48:23,867 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:48:33,070 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:50:31,457 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 18:50:50,979 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:51:00,370 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:03,069 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:10,365 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:11,519 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 18:52:13,428 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:20,537 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:30,018 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:33,119 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:40,466 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:43,597 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:51,036 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:52:53,117 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:53:00,403 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:53:10,665 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:53:20,034 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:54:40,950 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:54:50,265 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:55:00,768 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:55:10,197 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:55:20,561 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:55:30,878 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:57:13,737 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:57:22,942 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 18:58:19,024 - root - INFO - main.py:63 - START SERVER
2020-12-15 18:58:19,700 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 18:58:19,739 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 18:58:19,748 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 18:58:20,287 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 18:58:20,332 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 18:59:31,419 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:00:21,446 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:00:52,086 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:11:41,773 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:15:01,172 - root - ERROR - Object2.py:62 - 'int' object is not iterable
2020-12-15 19:15:03,310 - root - ERROR - Object2.py:856 - (192.168.0.201, 4370)--GetDeviceDataCount() get error code -2
2020-12-15 19:15:04,344 - root - WARNING - gateway_server.py:106 - equipment (192.168.0.201, 4370) is offline
2020-12-15 19:15:06,973 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 19:15:07,034 - root - ERROR - Object2.py:856 - (192.168.0.201, 4370)--GetDeviceDataCount() get error code -6
2020-12-15 19:15:07,319 - root - WARNING - gateway_server.py:250 - fail to connect to server ('192.168.0.117', 26)
2020-12-15 19:15:07,332 - root - WARNING - gateway_server.py:106 - equipment (192.168.0.201, 4370) is offline
2020-12-15 19:15:07,357 - root - WARNING - gateway_server.py:250 - fail to connect to server ('192.168.0.117', 23)
2020-12-15 19:15:11,202 - root - ERROR - Object2.py:62 - 'int' object is not iterable
2020-12-15 19:15:21,175 - root - ERROR - Object2.py:62 - 'int' object is not iterable
2020-12-15 19:15:30,475 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 19:15:30,483 - root - ERROR - Object2.py:856 - (192.168.0.201, 4370)--GetDeviceDataCount() get error code -6
2020-12-15 19:15:30,505 - root - WARNING - gateway_server.py:250 - fail to connect to server ('192.168.0.117', 26)
2020-12-15 19:15:30,520 - root - WARNING - gateway_server.py:250 - fail to connect to server ('192.168.0.117', 23)
2020-12-15 19:15:31,154 - root - ERROR - Object2.py:62 - 'int' object is not iterable
2020-12-15 19:15:31,503 - root - WARNING - gateway_server.py:106 - equipment (192.168.0.201, 4370) is offline
2020-12-15 19:15:41,159 - root - ERROR - Object2.py:62 - 'int' object is not iterable
2020-12-15 19:15:51,191 - root - ERROR - Object2.py:62 - 'int' object is not iterable
2020-12-15 19:15:52,176 - root - WARNING - gateway_server.py:106 - equipment (192.168.0.97, 26) is offline
2020-12-15 19:15:53,555 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 19:15:53,567 - root - ERROR - Object2.py:856 - (192.168.0.201, 4370)--GetDeviceDataCount() get error code -6
2020-12-15 19:15:53,629 - root - WARNING - gateway_server.py:250 - fail to connect to server ('192.168.0.117', 26)
2020-12-15 19:15:53,649 - root - WARNING - gateway_server.py:250 - fail to connect to server ('192.168.0.117', 23)
2020-12-15 19:15:54,278 - root - WARNING - gateway_server.py:106 - equipment (192.168.0.201, 4370) is offline
2020-12-15 19:33:01,723 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:37:11,533 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:38:01,917 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:40:41,737 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:49:21,566 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 19:57:31,727 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:02:01,600 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:03:51,477 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:08:41,779 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:13:32,000 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:17:21,232 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:18:31,788 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:22:31,769 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:28:51,209 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:31:02,104 - root - INFO - main.py:63 - START SERVER
2020-12-15 20:31:02,787 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2020-12-15 20:31:02,815 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 20:31:02,826 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 23))已成功连接。。online
2020-12-15 20:31:02,857 - root - INFO - gateway_server.py:242 - 服务端(('192.168.0.117', 26))已成功连接。。online
2020-12-15 20:31:02,905 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 23))已成功连接。。online
2020-12-15 20:31:02,936 - root - INFO - gateway_server.py:292 - 客户端(('192.168.0.97', 26))已成功连接。。online
2020-12-15 20:33:53,470 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:34:02,748 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:34:13,107 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:34:23,576 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:34:43,181 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:34:52,505 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:35:02,812 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:35:10,011 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:35:13,110 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:35:20,415 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:37:33,453 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:37:42,746 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:38:33,531 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:38:42,964 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:39:02,749 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:39:13,139 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:39:52,640 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:40:02,937 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:43:21,307 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:45:12,783 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:45:20,271 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:45:23,446 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:45:30,772 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:45:43,204 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:45:50,524 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:45:52,571 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:46:00,844 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:46:30,942 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:46:33,035 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:46:40,244 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:46:43,437 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:47:10,738 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:47:20,207 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:47:32,580 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:47:40,914 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:47:43,051 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:47:50,474 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:48:01,933 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:50:11,230 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 20:50:13,218 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:50:22,543 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:50:32,787 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:50:43,056 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:51:13,344 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:51:22,696 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:51:33,056 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:51:42,577 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:51:52,940 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:52:03,324 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:52:12,595 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:53:02,827 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:53:13,181 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:57:13,359 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:57:20,749 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:57:22,794 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:57:31,060 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:58:12,721 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:58:21,015 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:58:23,015 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:58:30,270 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:58:33,351 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:58:42,724 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:59:33,362 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:59:40,708 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:59:42,786 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 20:59:50,065 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:00:02,549 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:00:13,038 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:00:32,759 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:00:43,158 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:01:02,892 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:01:13,206 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:01:42,498 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:01:52,973 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:03:02,810 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:03:13,135 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:04:20,394 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:04:30,798 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:06:40,363 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:06:50,666 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:08:03,757 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:08:11,037 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:08:13,132 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:08:20,432 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:09:10,187 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:09:20,493 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:09:32,992 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:09:43,334 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:10:00,978 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:10:10,126 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:10:20,560 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:10:30,997 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:10:53,027 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:11:00,276 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:11:03,390 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:11:10,708 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:11:23,128 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:11:32,566 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:11:53,157 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:12:02,480 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:12:41,513 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:13:03,086 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:13:12,539 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:13:23,023 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:13:42,681 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:16:31,249 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:19:43,217 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:19:50,587 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:19:52,691 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:20:01,005 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:20:42,943 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:20:50,221 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:20:52,285 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:21:00,642 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:21:51,816 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:21:52,305 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:22:02,042 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:22:02,676 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:23:53,310 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:24:02,791 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:24:52,991 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:25:02,204 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:25:53,262 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:26:02,648 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:27:21,485 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:27:41,001 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:27:50,318 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:30:01,261 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:31:31,739 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:31:32,903 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:31:42,216 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:31:52,748 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:32:03,140 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:32:10,469 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:32:12,549 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:32:20,857 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:33:10,557 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:33:11,547 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 21:33:20,880 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:33:42,604 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:33:53,083 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:34:53,136 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:35:02,421 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:35:23,278 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:35:32,661 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:35:42,903 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:35:53,211 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:36:02,578 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:36:13,005 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:36:23,112 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:36:32,368 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:38:02,425 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:38:12,846 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:41:20,072 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:41:30,384 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:43:00,349 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:43:10,719 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:43:40,774 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:43:50,110 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:44:20,389 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:44:22,430 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:44:30,730 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:44:32,789 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:46:02,444 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:46:12,827 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:46:23,110 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:47:02,561 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:47:12,933 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:47:22,309 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:47:32,620 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:48:10,139 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:48:13,210 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:48:20,594 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:48:22,666 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:49:22,865 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:49:33,116 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:54:40,925 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:54:50,363 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:55:00,903 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:55:10,246 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:55:40,447 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:55:50,874 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:56:52,264 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:57:00,576 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:57:02,695 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:57:10,888 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:57:32,456 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:57:40,873 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:57:43,019 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:57:50,196 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:58:13,185 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:58:22,821 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:58:52,949 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:59:02,232 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:59:30,066 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:59:33,174 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:59:40,458 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 21:59:42,491 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:01:11,993 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 22:01:31,761 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 22:05:52,283 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:06:00,719 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:06:02,879 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:06:10,037 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:06:40,115 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:06:50,697 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:07:02,955 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:07:12,177 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:07:21,587 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 22:08:22,884 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:08:33,127 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:08:42,457 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:08:52,944 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:10:32,943 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:10:42,422 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:11:12,535 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:11:22,808 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:11:52,821 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:12:02,202 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:15:51,563 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 22:16:00,733 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:16:11,014 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:16:13,189 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:16:20,619 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:16:22,725 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:16:30,960 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:16:42,329 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:16:52,613 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:17:00,972 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:17:10,352 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:18:43,033 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:18:52,371 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:19:02,948 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:19:22,707 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:19:33,039 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:19:43,228 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:19:52,486 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:20:02,824 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:20:13,086 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:20:32,659 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:20:43,043 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:20:51,593 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 22:20:52,484 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:21:22,510 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:21:42,468 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:21:52,717 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:22:03,161 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:22:12,264 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:22:22,490 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:24:30,182 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:24:40,631 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:27:20,471 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:27:31,041 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:28:00,159 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:28:10,459 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:28:20,682 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:28:42,444 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:28:52,882 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:29:00,061 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:29:03,161 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:29:10,431 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:29:42,501 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:29:52,918 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:30:13,322 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:30:22,526 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:31:50,941 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:32:00,221 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:32:51,263 - root - ERROR - Object2.py:62 - 'int' object has no attribute 'hex'
2020-12-15 22:32:52,291 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:33:02,702 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:33:13,144 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:33:22,334 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:33:52,537 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:34:02,816 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:37:10,873 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:37:20,083 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:37:30,491 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:37:40,836 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:37:41,912 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:38:00,487 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:38:33,498 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:38:57,546 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:00,710 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:10,078 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:20,429 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:30,878 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:32,942 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:40,338 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:42,419 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:50,751 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:39:52,834 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:40:03,123 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
2020-12-15 22:40:33,180 - root - ERROR - task_controler.py:170 - [Errno 32] Broken pipe
